### PR TITLE
support shallow copy of pipelines

### PR DESCRIPTION
**Pull request by michalozeryflato**

### CODE REVIEW

**Comment by michalozeryflato** [comment]
made this change to allow shallow copy & extended of dynamic pipeline  (adding augmentation during train)

File: fuse/data/pipelines/pipeline_default.py [hunk 1]
```diff
@@ -54,6 +56,12 @@ def __init__(
             self._op_ids = op_ids
         self._verbose = verbose
 
+    def copy(self):
```
Comment (mosheraboh) [inline fuse/data/pipelines/pipeline_default.py:59]: Nice,
Can you please add in a comment that it is a shallow copy (both pipelines will still use the same instances of ops)?

**Review by mosheraboh — APPROVED** [review]
Nice
Minor comment inline

File: fuse/data/pipelines/pipeline_default.py (hunk 1 above)
Comment (michalozeryflato) [reply fuse/data/pipelines/pipeline_default.py:59]: done

**Review by mosheraboh — APPROVED** [review]

**Pull request merged**